buildpipeline/pipelines.json

```diff
@@ -425,7 +438,42 @@
             "Type": "build/product/",
             "PB_BuildType": "Release"
           }
-        }
+        },
+        {
+          "Name": "Dotnet-CoreClr-Trusted-BuildTests",
+          "Parameters": {
+            "HelixJobType": "test/functional/cli/",
+            "TargetsWindows": "false",
+            "Rid": "linux-x64",
```

Comment:
Change this as well to "rhel6-x64" (make sure the "rhel6" part matches the change you made to the RID in the product build)